src/components/MenuPanel/MenuPanel.js

```diff
@@ -29,18 +29,18 @@ function MenuPanel({
 }) {
   const { apps } = useOrganizationState()
   const history = useHistory()
+  const location = useLocation()
   const [activeApp, setActiveApp] = useState()
 
   useEffect(() => {
-    const parts = history.location.pathname.split('/')
+    const parts = location.pathname.split('/')
```

Comment:
ah nice, can tou tell me in which case was not working? since i couldnt reproduce it

Reply:
It wasn't working at all for me :thinking:, as if the `useEffect` hook wasn't listening correctly to the `history` changes